connectors/src/connectors/webcrawler/index.ts

```diff
@@ -326,3 +358,45 @@ export async function retrieveWebCrawlerObjectsParents(
 
   return new Ok(parents);
 }
+
+type GetWebCrawlerConfigurationResBody =
+  WithConnectorsAPIErrorReponse<WebCrawlerConfigurationType>;
+
+async function _getWebcrawlerConfiguration(
+  req: Request<{ connector_id: string }, GetWebCrawlerConfigurationResBody>,
+  res: Response<GetWebCrawlerConfigurationResBody>
+) {
+  const connector = await ConnectorModel.findByPk(req.params.connector_id);
```

Comment:
I merged my branch, you'll need to update to:
```suggestion
  const connector = await ConnectorResource.fetchById(req.params.connector_id);
```